src/connection_string.ts

```diff
@@ -1090,6 +1090,9 @@ export const OPTIONS = {
     target: 'tls',
     type: 'boolean'
   },
+  timeoutMS: {
+    type: 'uint'
+  },
```

Comment:
Also fine with this, just bus factor++ we may want to assert more about this value than just unsigned int. We're building this functionality on `setTimeout` which has an upper limit.

```txt
(node:65848) TimeoutOverflowWarning: 2147483648 does not fit into a 32-bit signed integer.
Timeout duration was set to 1.
```
Node decides this is a warning which will manifest as a timeout of 1ms but potentially we want to consider this an error before even creating the timeout.

Reply:
I think we can leave that for later when we actually implement it. This was just to get through the unified tests.